### PR TITLE
Improve project

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Improve project
| Type?         | improvement
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | N/A
| How to test?  | https://github.com/PrestaShop/welcome/pull/67#issuecomment-700193711

### CODE REVIEW

**Review by jolelievre — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Comment by Progi1984** [comment]
@sarahdib It's open for QA ;)

**Review by sowbiba — DISMISSED** [review]

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    'import',
+  ],
+  extends: [
+    'prestashop',
+  ],
+  globals: {
+    PS_LAYERED_INDEXED: true,
+    filters: true,
+    prestashop: true,
+    translations: true,
```
Comment (PierreRambaud) [inline .eslintrc.js:23]: Not sure these variables are needed.

Comment (Progi1984) [reply .eslintrc.js]: Fixed

**Review by PierreRambaud — DISMISSED** [review]

**Review by sowbiba — DISMISSED** [review]

**Comment by sarahdib** [comment]
Hello @Progi1984 

Ok with a fresh install the module is display.

But there is a regression with this PR

1. Stop on boarding
2. Uninstall module welcome
3. Install module welcome
4. Go to the dashboard
5. See the module is not display anymore

**Review by PierreRambaud — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @sarahdib

**Pull request merged**